### DISCUSSION TITLE
Welcome to EMQ X Discussions!

### DISCUSSION THREAD

**Discussion by Rory-Z** [emqx/emqx#3887] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zmstone** [comment]
Zaiming (Stone) Shi here.
I am the product owner of EMQ X.
Together with @tigercl and @emqx/emqx-devs team, I make sure issues, PRs and discussions are well prioritised/followed up.
For feature and design proposals, you are welcome to add discussions and pull requests here https://github.com/emqx/eip
